CodeConverter/CSharp/TypeConversionAnalyzer.cs

```diff
@@ -333,6 +340,10 @@ private bool IsExplicitNumericConversion(VBSyntax.ExpressionSyntax vbNode)
         if (vbConversion.IsNumeric && (vbType.IsEnumType() || vbConvertedType.IsEnumType())) {
             return TypeConversionKind.NonDestructiveCast;
         }
+
+        if (vbConversion.IsNarrowing && vbConversion.IsString && vbConversion.ToString().Contains("InvolvesEnumTypeConversions")) {
```

Comment:
Shame that Roslyn doesn't expose some flags about the conversions.
Need to find a better way to check this.

Reply:
Yeah the object contains more flags than it exposes. Looking at the roslyn codebase around that internal flag might yield a hint